CHANGES.md

```diff
@@ -18,19 +18,24 @@ To be released.
     -  `ValidatorSetStateCompleters`
  -  Removed type parameter `T` for `IAction` from `ActionEvaluator` and
     `IBlockChainStates`.  [[#2703]]
+ -  `ActionEvaluator<T>()` constructor now explicitly requires a new
+    `feeCalculator` parameter.  [[#2566]]
```

Comment:
Because this pull request opened a long time ago, this line seems staled. (ref #2703)
```suggestion
 -  `ActionEvaluator()` constructor now explicitly requires a new
    `feeCalculator` parameter.  [[#2566]]
```